account_invoice_search_by_reference/models/account_invoice.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class AccountInvoice(models.Model):
```

Comment:
In v13, there is no longer file named "accont_invoice.py", it should be account_move.py.
Also no `class AccountInvoice`, should be `class AccountMove`

Reply:
@newtratip please fix it.

Reply:
@newtratip can you attend this?

Reply:
Fixed, please review again.

Thanks.